libcaf_net/caf/net/stream_transport.hpp

```diff
@@ -166,7 +163,7 @@ class stream_transport : public stream_transport_base<Application> {
     return false;
   }
 
-  void write_packet(id_type, span<buffer_type*> buffers) override {
+  void write_packet(id_type, span<byte_buffer*> buffers) override {
```

Comment:
`caf/byte_buffer.hpp` is not included in this file.

Reply:
Here too.. I thought `caf/fwd.hpp` will suffice?

Reply:
We are using the buffers in the function body. For that, the compiler needs to know member functions and memory layout. It only works, because some other headers pull in `byte` and `vector` (and the compiler knows from the `fwd.hpp` that `byte_buffer` is an alias type).